From c7e216b61ef58c0fdc701ef860d567f6932be786 Mon Sep 17 00:00:00 2001
From: TheAtt1la <alittauwu@gmail.com>
Date: Sat, 4 Feb 2023 02:09:37 +0100
Subject: [PATCH 2/2] Revert Don't retry kill if cgroups isn't available

---
 libprocessgroup/processgroup.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/libprocessgroup/processgroup.cpp b/libprocessgroup/processgroup.cpp
index fcc279e55..bdda1020c 100644
--- a/libprocessgroup/processgroup.cpp
+++ b/libprocessgroup/processgroup.cpp
@@ -451,11 +451,6 @@ static int KillProcessGroup(uid_t uid, int initialPid, int signal, int retries,
             *max_processes = processes;
         }
         LOG(VERBOSE) << "Killed " << processes << " processes for processgroup " << initialPid;
-        if (!CgroupsAvailable()) {
-            // makes no sense to retry, because there are no cgroup_procs file
-            processes = 0;  // no remaining processes
-            break;
-        }
         if (retry > 0) {
             std::this_thread::sleep_for(5ms);
             --retry;
-- 
2.34.1


